python/social_bot/envs/base.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2019 Horizon Robotics. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import gym
+
+import social_bot.pygazebo as gazebo
+
+
+class EnvBase(gym.Env):
```

Comment:
Is "base.py" dedicated to implement gazebo rendering? If so, maybe we could change the file name to something like "render_base.py" or "gazeboenv_base.py" .

Reply:
Changed.